include/thread_pool/thread_pool.hpp

```diff
@@ -150,17 +186,17 @@ namespace tp
     {
         const auto ok = try_post(std::forward<Handler>(handler));
         assert(ok);
+        _tp_unused(ok);
     }
 
     template <typename TSettings>
-    inline Worker& ThreadPool<TSettings>::getWorker()
+    inline typename ThreadPool<TSettings>::FixedWorker& ThreadPool<TSettings>::getWorker()
```

Comment:
Aren't we targeting C++14 though? If you feel like this should target C++11 *(to reach more people)* we should discuss about that and eventually change the README. I like targeting the newest standard because it drives people to update their toolchains and to learn the new features/idioms. Thoughts, @inkooboo?